### DISCUSSION TITLE
OptiSim and MaxMin

### DISCUSSION THREAD

**Discussion by alnaba1** [theochem/Selector#119] [General]
As requested by @FanwangM, this discussion post is to mention the differences and similarities between OptiSim and MaxMin for the manuscript. 

The way OptiSim works is by choosing a random subset of points with length k, then selecting the point from that random subset that has the maximum minimum (MaxMin) distance to previously selected points, and then eliminating all candidates that are closer than allowed by the radius. This means that when an appropriate radius is selected and k approaches infinity (greater than the number of possible candidates), the random subset will consist of all possible candidates which is equivalent to doing the MaxMin algorithm.

One of the testers for OptiSim confirms that equivalent results are obtained to MaxMin when k is chosen to be larger than the number of candidates.

**Comment by FanwangM** [comment]
This is good to have. Thank you for the input.

**Reply by PaulWAyers** [reply]
This information should be added to documentation for OptiSim (and for the package in general, perhaps in the web site).
